File: 0217-contains-duplicate/0217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        int a= nums.size();
        for (int i= 1; i<a; i++){
            if (nums[i-1]==nums[i]){
                return true;
            }
        }
        return false;
    }
};

File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums: List[int]) -> bool:
        a= nums.sort()
        b= len(nums)
        for i in range (1, b):
            if (nums[i-1]==nums[i]):
                return True
            else:
                i+=1
        return False